### PR TITLE
Skin world traveler

### PR DESCRIPTION
**Pull request by mikeyrayvon**
also fixes background animation init on supertap YEA!

### CODE REVIEW

File: cordova/www/games/worldtraveler/worldtraveler.js [hunk 1]
```diff
@@ -1,21 +1,31 @@
 var Worldtraveler = {
   introDialog: [
-    "U gotta travel the distance between Templo Mayor and Piramide del Sol",
-    "thats like a lot",
-    "much traveler points woop woop",
+    "OK " + Game.getUsername() + ".... it's time for WORLD TRAVELER..!!",
+    "The game where U gotta travel the distance between two famous pyramids...",
```
Comment (varffs) [inline cordova/www/games/worldtraveler/worldtraveler.js]: surely `at least the distance`

**Comment by varffs** [comment]
seems a bit unfair that you only get 1 point for traveling a km. maybe we should give you a point for every 10m or every meter. or hilariously convert the km value into feet and get 1 point every yard?

File: cordova/www/games/worldtraveler/worldtraveler.js [hunk 2]
```diff
@@ -1,21 +1,31 @@
...
+    "Since u love traveling, " + Game.getUsername() + "... let's go for a walk...!!!",
   ],
   finalFailDialog: [
-    "Nah, u didnt travel enough",
-    "ill set ur actual position as initial position for the next time",
+    "Nah, u didnt travel enough, " + Game.getUsername() + "...",
+    "I'll set ur new position as the starting line for the next time...",
+    "........",
+    "........",
+    "OK! Back to walking...see ya next round...",
```
Comment (varffs) [inline cordova/www/games/worldtraveler/worldtraveler.js]: `in the next world`?

**Comment by varffs** [comment]
Its all good with the skin etc but I'm not sure bout the dialog. Its kinda confusing to say travel as the game is about being xKm away from where you were last time you played, not about the distance you have travelled since you last played. I think that needs revision. Happy to do it if you want?

**Comment by mikeyrayvon** [comment]
@varffs yea go for it ! dialog and points

**Pull request merged**